marlowe-dashboard-client/src/Contract/View.purs

```diff
@@ -20,7 +23,16 @@ contractsScreen :: forall p. ContractStatus -> HTML p Action
 contractsScreen contractStatus =
   div_
     [ h2_
-        [ text "Dashboard home" ]
+        [ text "Home" ]
+    , a
+        [ classNames Css.fixedPrimaryButton
```

Comment:
this seems like a bad idea, Halogen has gone to the trouble of newtyping class names and here we just ignore them. It's certainly nice to have `classNames` for when we are using strings in line to avoid `classes <<< map ClassName $ [...]` but when we are importing classes defined in a different module (`CSS`) it is a different story.

Reply:
Eventually I agree, for the moment I think it would complicate things. I imagine the helpers like `applyWhen` to be thought to use with some `Css.something <> ["p-2"]` etc.